src/internal/connector/onedrive/collection_test.go

```diff
@@ -611,6 +629,12 @@ func (suite *GetDriveItemUnitTestSuite) TestDownloadContent() {
 
 	itemWID.SetId(ptr.To("brainhooldy"))
 
+	m := &mockURLCache{
+		Get: func(ctx context.Context, itemID string) (itemProps, error) {
```

Comment:
Seems like this could be the default behavior for a `NewMockURLCache`, instead of needing the caller to add it, and risk a nil pointer failure as a gotcha.

Reply:
I'll move this to mock folder in a later PR